sdk/samples/core/README.md

```diff
@@ -127,7 +120,10 @@ Set the following environment variables for all samples:
     ```bash
     mkdir build
     cd build
-    cmake -DAZ_MQTT_TRANSPORT_IMPL=MOSQUITTO -DAZ_PLATFORM_IMPL=POSIX ..
+    # for mosquitto samples
+    cmake -DAZ_MQTT_TRANSPORT_IMPL=MOSQUITTO -DAZ_PLATFORM_IMPL=POSIX -DCMAKE_TOOLCHAIN_FILE=<path_to_vcpkg_repo>/scripts/buildsystems/vcpkg.cmake ..
+    # for paho samples
+    cmake -DAZ_MQTT_TRANSPORT_IMPL=PAHO -DAZ_PLATFORM_IMPL=POSIX -DCMAKE_TOOLCHAIN_FILE=<path_to_vcpkg_repo>/scripts/buildsystems/vcpkg.cmake ..
     ```
 2. Compile and run the sample.
 
```

Comment:
Would be good to mention here that you can edit MQTT configuration default values in `az_mqtt_config.h` (like the port number for example)

Reply:
Added in the set up section!